it/mail/chrome/messenger/importMsgs.properties

```diff
@@ -280,78 +280,78 @@
-ImportSettingsListLabel=Select application to import settings from:
+ImportMailDialogTitle=Importa e-mail
+ImportAddressBooksDialogTitle=Importa rubriche
+ImportSettingsDialogTitle=Importa impostazioni
+ImportMailListLabel=Seleziona applicazione da cui importare le e-mail:
+ImportAddressBooksListLabel=Seleziona formato rubrica:
+ImportSettingsListLabel=Seleziona applicazione da cui importare le impostazioni:
 
 #Error strings
-ImportAlreadyInProgress=È in corso una operazione di importazione. Riprovare non appena l'importazione corrente è terminata.
+ImportAlreadyInProgress=È in corso una operazione di importazione. Riprova non appena l'importazione corrente è terminata.
 
 #Error strings for settings import
 ImportSettingsBadModule=Impossibile caricare il modulo delle impostazioni
-ImportSettingsNotFound=Impossibile trovare le impostazioni. Assicurarsi che il programma sia installato su questo computer.
-ImportSettingsFailed=Si è verificato un errore nell'importazione delle impostazioni. Parte, o tutte le impostazioni possono non essere state importate.
+ImportSettingsNotFound=Impossibile trovare le impostazioni. Controlla per assicurarti che il programma sia installato su questo computer.
+ImportSettingsFailed=Si è verificato un errore nell'importazione delle impostazioni. Alcune, o tutte, le impostazioni possono non essere state importate.
 # LOCALIZATION NOTE : Do not translate the word "%S" below.
 ImportSettingsSuccess=Le impostazioni sono state importate da %S
 
 #Error string for mail import
-ImportMailBadModule=Impossibile caricare il modulo di importazione posta
-ImportMailNotFound=Impossibile trovare posta da importare. Assicurarsi che il programma di posta sia correttamente installato su questo computer.
-ImportEmptyAddressBook=Impossibile importare una rubrica vuota (%S).
+ImportMailBadModule=Impossibile caricare il modulo di importazione e-mail
+ImportMailNotFound=Impossibile trovare posta da importare. Assicurati che l'applicazione e-mail sia correttamente installata su questo computer.
+ImportEmptyAddressBook=Impossibile importare la rubrica vuota (%S).
 # LOCALIZATION NOTE : Do not translate the word "%S" below.
-ImportMailFailed=Si è verificato un errore durante l'importazione della posta da %S
+ImportMailFailed=Si è verificato un errore durante l'importazione e-mail da %S
 # LOCALIZATION NOTE : Do not translate the word "%S" below.
-ImportMailSuccess=La posta è stata importata con successo da %S
+ImportMailSuccess=E-mail importate con successo da %S
 
 # Error string for address import
 ImportAddressBadModule=Impossibile caricare il modulo di importazione rubriche.
-ImportAddressNotFound=Impossibile trovare rubriche da importare. Assicurarsi che il programma o il formato scelto sia correttamente installato su questo computer.
+ImportAddressNotFound=Impossibile trovare rubriche da importare. Controlla per assicurarti che l'applicazione o il formato scelti siano correttamente installati su questo computer.
```

Comment:
CORRECTION NEEDED. 

I would shorten the second sentence by replacing "Controlla per assicurarti" with "Assicurati".